src/components/TimeControl/util.ts

```diff
@@ -433,7 +433,7 @@ export function usedForCheating(time_control) {
         case "byoyomi":
             return !(
                 time_control.main_time > QUESTIONABLE_ABSOLUTE_TIME ||
-                time_control.period_time > QUESTIONABLE_SECONDS_PER_MOVE
```

Comment:
Isn't this going to flag pretty much every live game as questionable? It's quite common to have say 5x30s byoyomi, which if i'm not mistaken, would be flagged as questionable here if the main time wasn't at least 15 minutes